elide-core/src/main/java/com/yahoo/elide/core/EntityBinding.java

```diff
@@ -240,6 +240,16 @@ public EntityBinding(EntityDictionary dictionary, Class<?> cls, String type, Str
      */
     private void bindEntityFields(Class<?> cls, String type, Collection<AccessibleObject> fieldOrMethodList) {
         for (AccessibleObject fieldOrMethod : fieldOrMethodList) {
+            // Special handling for Eclipse. Load missing annotations
+            if (fieldOrMethod.getAnnotations().length == 0 && fieldOrMethod instanceof Method) {
```

Comment:
Can you explain a bit more what this is about?

Reply:
Moved to #1909